src/main/java/io/burt/athena/result/s3/InputStreamResponseTransformer.java

```diff
@@ -28,12 +30,15 @@
     private Throwable error;
     private AtomicBoolean complete;
     private AtomicInteger approximateBufferSize;
+    private AtomicInteger requests;
```

Comment:
I guess this should be an `AtomicLong` to match the type used in the subscription?

Reply:
Given that we limit it to 1000, an integer should be perfectly fine.